examples/core/instancing/app.js

```diff
@@ -158,11 +159,15 @@ class AppAnimationLoop extends AnimationLoop {
   }
 
   onRender(animationProps) {
-    const {gl, framebuffer, useDevicePixels, _mousePosition, statsWidget} = animationProps;
+    const {gl, statsWidget} = animationProps;
 
     statsWidget.update();
 
+    /* TODO - picking temporarily disabled
```

Comment:
What happened here?

Reply:
`pickModels` depends on knowledge of scenegraph `GroupNode` which is no longer part of core.

`pickModels` was already broken, and this adds to the effort required to fix it we need to make a decision on whether to temporarily cut picking in luma.gl or invest in restoring it.